### PR TITLE
Alert sound option.  Defaults to off.  Default sound is Limmy: "HO".

### PR DESCRIPTION
**Pull request by Gantonator**
Plays a sound when the !alert command is used.

Can be turned on in `settings.js` by setting `isAlertSound = true;`
The sound can be set to whatever you like: either by swapping out `alert.mp3` or adding a new file and changing `const alertSoundFile = "assets/sounds/alert.mp3";` in `settings.js`.

The default sound is from here https://clips.twitch.tv/CheerfulAverageJamSuperVinlin.

Please credit `boglinjones` from the discord as they suggested the sound.

GDNWIlQ

### CODE REVIEW

File: settings.js [hunk 1]
```diff
@@ -13,4 +13,10 @@ const spotlightEmoji = '⭐';
 // Allow twitch emotes, can be true or false.
 const twitchEmotes = true;
 
+// Play a sound when the "!alert" command is used
+const isAlertSound = false;
```
Comment (FunkotronKing) [inline settings.js:17]: Just so variable make sense could this be 'playAlertSound?'

File: src/handlers.js [hunk 2]
```diff
@@ -9,6 +9,7 @@ actionHandlers['!alert'] = {
     handle: (context, textContent) => {
         const formattedText = popup.formatEmotes(textContent, context.emotes, true).substr(7);
         popup.showText(formattedText, alertBg);
+        if (isAlertSound) new Audio(alertSoundFile).play();
```
Comment (FunkotronKing) [inline src/handlers.js:12]: Can we use the long form if statement? they are far easier to read for people newer to coding

Comment (Gantonator) [reply src/handlers.js]: Done!

File: settings.js (hunk 1 above)
Comment (Gantonator) [reply settings.js]: Done!

**Pull request merged**